src/Constants.ts

```diff
@@ -31,10 +35,20 @@ export enum LANGUAGE {
 	TYPESCRIPT = 'typescript'
 }
 
+export enum OUTPUT_FORMAT {
+	CSV = 'csv',
+	HTML = 'html',
+	JSON = 'json',
+	JUNIT = 'junit',
+	TABLE = 'table',
+	XML = 'xml'
+}
+
 export const Services = {
 	Config: "Config",
 	RuleManager: "RuleManager",
 	RuleEngine: "RuleEngine",
+	DependencyChecker: "DependencyChecker",
```

Comment:
Should we alpha sort these?

Reply:
I hadn't thought to do that, but now that you've pointed it out, yes, we absolutely should. I'll do that.